### DISCUSSION TITLE
CocoaRestClient

### DISCUSSION THREAD

**Discussion by AndroidKitKat** [elrumo/macOS_Big_Sur_icons_replacements#1109] [Icon request]
Curious if anyone could whip up an icon for a somewhat obscure app.

http://mmattozzi.github.io/cocoa-rest-client/

**Comment by dursonATWIT** [comment]
Threw together a quick design for you, I submitted it but might take a little for it to be approved

![](https://i.imgur.com/zJR9YYQ.png)
